MonoGame.Framework/MathF.cs

```diff
@@ -0,0 +1,71 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+
+namespace Microsoft.Xna.Framework
+{
+#if (!NETCOREAPP && !NETSTANDARD2_1) || NETCOREAPP1_0 || NETCOREAPP1_1
```

Comment:
dotnet 5 sets both `NETCOREAPP` and `NET5_0` (I just tested this using RC2). I don't expect them to come with anymore net standards as that was the whole point of dotnet 5. The idea is that all platforms will now support dotnet 5, instead of having netstandard as a lowest common denominator / interface / spec.

I also do not really like that preprocessor check, but I could not really come up with a much better version.

Reply:
https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/#for-the-impatient-tldr third paragraph;
> We won’t be releasing a new version of .NET Standard, but .NET 5 and all future versions will continue to support .NET Standard 2.1 and earlier.

Would be ideal if every MG project moved to NET5 but that is a far-away future, so it's understandable if some projects stay on netstandard20 or lower. These "legacy" projects should probably include `MathF` as a `Compile` item instead as mentioned.

Reply:
To make the if simpler i could also remove the checks for dotnet core 1.0 and 1.1 as no one will use those. They are past end of support from Microsoft. I just added them for good measure, so that it would not break if compiled with it.

Reply:
Okay, cool then!